packages/shared/src/lib/flows/flow-operations.ts

```diff
@@ -241,7 +238,7 @@ export const FlowOperationRequest = Type.Union([
   Type.Object(
     {
       type: Type.Literal(FlowOperationType.UPDATE_TRIGGER),
-      request: UpdateTriggerRequest,
+      request: TriggerWithOptionalId,
     },
```

Comment:
there is no add trigger request, both add and update are via this flow operation